src/uniprotkb/components/protein-data-views/__tests__/XrefView.spec.tsx

```diff
@@ -1,26 +1,28 @@
-import { render } from '@testing-library/react';
+import { screen, render } from '@testing-library/react';
 
 import XRefView, {
   getPropertyString,
   processUrlTemplate,
-  getPropertyLink,
+  getPropertyLinkAttributes,
   getDatabaseInfoAttribute,
 } from '../XRefView';
 
-import { XrefUIModel } from '../../../utils/xrefUtils';
 import { PropertyKey } from '../../../types/modelTypes';
 import { DatabaseInfoPoint } from '../../../types/databaseRefs';
 
-import xrefUIData from './__mocks__/xrefUIData.json';
+import xrefs from './__mocks__/xrefUIData';
 
 describe('XRefView', () => {
   test(`should render section`, () => {
-    const xrefs = xrefUIData as XrefUIModel[];
     const { asFragment } = render(
-      <XRefView xrefs={xrefs} primaryAccession="P01234" />
+      <XRefView xrefs={xrefs.normal} primaryAccession="P01234" />
```

Comment:
```suggestion
      <XRefView xrefs={xrefs.standard} primaryAccession="P01234" />
```
I think there was an issue here because the corresponding snapshot had become empty